ui/src/discovery.ts

```diff
@@ -1,133 +1,12 @@
-    shareableEntityIdentifier:
-      "rad:git:hwd1yre85ddm5ruz4kgqppdtdgqgqr4wjy3fmskgebhpzwcxshei7d4ouwe",
-    stats: {
-      commits: 220,
-      branches: 12,
-      contributors: 4,
-    },
-    maintainers: [
-      {
-        handle: "rafalca",
-        avatar: {
-          background: {
-            r: 255,
-            g: 0,
-            b: 179,
-          },
-          emoji: "🐬",
-        },
-      },
-    ],
-    // TODO(sos): make sure tracking info is part of project feed api
-    tracked: false,
-  },
-  {
-    domain: "radicle",
-    name: "radicle-surf",
-    description: "A nice app for nice things",
-    shareableEntityIdentifier:
-      "rad:git:hwd1yre85ddm5ruz4kgqppdtdgqgqr4wjy3fmskgebhpzwcxshei7d4ouwe",
-    stats: {
-      commits: 220,
-      branches: 12,
-      contributors: 4,
-    },
-    maintainers: [
-      {
-        handle: "rafalca",
-        avatar: {
-          background: {
-            r: 50,
-            g: 168,
-            b: 82,
-          },
-          emoji: "🐎",
-        },
-      },
-    ],
-    tracked: true,
-  },
-  {
-    domain: "radicle",
-    name: "radicle-link",
-    description: "A nice app for nice things",
-    shareableEntityIdentifier:
-      "rad:git:hwd1yre85ddm5ruz4kgqppdtdgqgqr4wjy3fmskgebhpzwcxshei7d4ouwe",
-    stats: {
-      commits: 220,
-      branches: 12,
-      contributors: 4,
-    },
-    maintainers: [
-      {
-        handle: "rafalca",
-        avatar: {
-          background: {
-            r: 88,
-            g: 50,
-            b: 168,
-          },
-          emoji: "🖕🏼",
-        },
-      },
-    ],
-    tracked: false,
-  },
-  {
-    domain: "radicle",
-    name: "radicle-registry",
-    description: "A nice app for nice things",
-    shareableEntityIdentifier:
-      "rad:git:hwd1yre85ddm5ruz4kgqppdtdgqgqr4wjy3fmskgebhpzwcxshei7d4ouwe",
-    stats: {
-      commits: 220,
-      branches: 12,
-      contributors: 4,
-    },
-    maintainers: [
-      {
-        handle: "rafalca",
-        avatar: {
-          background: {
-            r: 221,
-            g: 255,
-            b: 28,
-          },
-          emoji: "👼",
-        },
-      },
-    ],
-    tracked: false,
-  },
-  {
-    domain: "radicle",
-    name: "radicle-long-repository-name-that-truncates-nicely",
-    description: "A nice app for nice things",
-    shareableEntityIdentifier:
-      "rad:git:hwd1yre85ddm5ruz4kgqppdtdgqgqr4wjy3fmskgebhpzwcxshei7d4ouwe",
-    stats: {
-      commits: 220,
-      branches: 12,
-      contributors: 4,
-    },
-    maintainers: [
-      {
-        handle: "rafalca",
-        avatar: {
-          background: {
-            r: 255,
-            g: 0,
-            b: 179,
-          },
-          emoji: "🚬",
-        },
-      },
-    ],
-    tracked: false,
-  },
-];
+import * as api from "./api";
+import * as project from "./project";
+import * as remote from "./remote";
+
+const feedStore = remote.createStore<project.Project[]>();
```

Comment:
If this is just holding projects it's fine to name it accordingly, we can move to a general feed store once we have proper feed items.